OrbitGl/LiveFunctionsDataView.h

```diff
@@ -24,6 +26,8 @@ class LiveFunctionsDataView : public DataView {
   void OnDataChanged() override;
   void OnTimer() override;
 
+  void JumpToBox(const TextBox* box) const;
```

Comment:
I think that you have to remove this code, maybe is something old one after merging.

Reply:
Thanks, this was just a left over. The implementation didn't exist anymore.